components/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import cn from "clsx";
+import type { ReactElement } from "react";
+import NetideeLogo from "../public/netidee.svg";
```

Comment:
@nikgraf that's the part I really don't want to do, I just put it in public for now, but that definitely needs to change

Reply:
still seems to be a reasonable and straight forward solution, let's move forward with it